### PR TITLE
Revert "💚 PIC-1326: Add missing helm templates (#7)"

### PR DESCRIPTION
**Pull request by katie-cawthorne**
This reverts commit 0261f10105300fff2f0daf213f5b95c71c38a200.

### CODE REVIEW

File: helm_deploy/court-list-splitter/Chart.yaml [hunk 1]
```diff
@@ -3,3 +3,11 @@ appVersion: "1.0"
...
 name: court-list-splitter
 version: 0.1.0
+
+dependencies:
+  - name: generic-service
+    version: 1.0.9
+    repository: https://ministryofjustice.github.io/hmpps-helm-charts
+  - name: generic-prometheus-alerts
+    version: 0.1.4
+    repository: https://ministryofjustice.github.io/hmpps-helm-charts
```
Comment (katie-cawthorne) [inline helm_deploy/court-list-splitter/Chart.yaml:13]: I noticed that CHER project also had this prometheus dependency so added this also as must come from the bootstrapping

File: helm_deploy/values-dev.yaml [hunk 2]
```diff
@@ -1,30 +1,11 @@
...
+generic-service:
+  springProfiles: sqs-read
+
+  ingress:
+    enabled: true
+    enable_whitelist: true
+    hosts:
+      - host: court-list-splitter-dev.hmpps.service.justice.gov.uk
+        cert_secret: court-probation-dev-cert-secret
+    path: /
```
Comment (katie-cawthorne) [inline helm_deploy/values-dev.yaml:10]: i added the values under the generic-service as this is what's needed to work with generic-service

File: helm_deploy/values-dev.yaml [hunk 3]
```diff
@@ -1,30 +1,11 @@
-
-minReplicaCount: 1
-maxReplicaCount: 2
-
-springProfiles: sqs-read
-
-image:
-  repository: quay.io/hmpps/court-list-splitter
-  tag: latest
-  port: 8080
```
Comment (katie-cawthorne) [inline helm_deploy/values-dev.yaml:10]: I removed values that are already held in Values.yaml, to remove repetition as they were repeated for each environment.

File: helm_deploy/values-preprod.yaml [hunk 4]
```diff
@@ -1,37 +1,18 @@
+generic-service:
+  springProfiles: ""
```
Comment (jhevans) [inline helm_deploy/values-preprod.yaml:2]: I can see this is the same as what was there previously but should be set to sqs-read, same for prod 🤔  Ah! [This PR](https://github.com/ministryofjustice/court-list-splitter/pull/18) should have been merged 😬  

Could you add that profile to preprod and prod as part of this deployment as that other PR will be stale now? I'll close the other one.

**Review by jhevans — CHANGES_REQUESTED** [review]

File: helm_deploy/values-preprod.yaml (hunk 4 above)
Comment (katie-cawthorne) [reply helm_deploy/values-preprod.yaml:2]: Yep sure no probs

**Review by jhevans — APPROVED** [review]

**Pull request merged**